app/assignment/directives/modals/commentModal/commentModal.css

```diff
@@ -8,5 +8,17 @@
 }
 
 .comment-modal__comment {
-  white-space: pre;
+  display: flex;
+	justify-content: space-between;
```

Comment:
Something funny happened here with indentation? File should be 2 spaces?

Reply:
I might have copy a line from somewhere and mixed up tabs and spaces. It's fixed to 2 spaces now.